src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -461,4 +463,24 @@ protected function isGrantedOr403($permission)
             throw new AccessDeniedHttpException();
         }
     }
+
+    protected function isApiRequest()
+    {
+        if (!$this->config->isApiRequest()) {
+            return false;
+        }
+
+        $bundles = array(
+            'JMSSerializerBundle',
+            'BazingaHateoasBundle'
+        );
+        $enabledBundles = $this->container->getParameter('kernel.bundles');
```

Comment:
I can do that, but don't want to put 'JMSSerializerBundle', 'BazingaHateoasBundle' many times in the code ...